images/macos/provision/core/powershell.sh

```diff
@@ -9,6 +9,10 @@ brew cask install powershell
 # A dummy call of `az` to initialize ~/.azure directory before the modules are installed
 az -v
 
+# Update PowerShellGet to eliminate download errors
+pwsh -Command "Install-Module -Name PowerShellGet -Force"
```

Comment:
Can we just add it to toolset?
https://github.com/actions/virtual-environments/blob/main/images/macos/toolsets/toolset-10.15.json#L113